x-pack/plugins/security_solution/public/common/components/visualization_actions/lens_embeddable.tsx

```diff
@@ -187,7 +187,7 @@ const LensEmbeddableComponent: React.FC<LensEmbeddableComponentProps> = ({
 
   const onFilterCallback = useCallback(
     async (e: ClickTriggerEvent['data'] | MultiClickTriggerEvent['data']) => {
-      if (!Array.isArray(e.data) || preferredSeriesType !== 'area') {
+      if (!isClickTriggerEvent(e) || preferredSeriesType !== 'area') {
```

Comment:
@elastic/security-solution looks like the cypress tests break if there's something modified in the directory security-solution and my specific changes don't break it. I ran the [test PR](https://github.com/elastic/kibana/pull/163229/files) with no changes and it broke too. Could you please take a look?